pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -73,6 +75,22 @@ const (
 	clusterImageSetFoundReason            = "ClusterImageSetFound"
 )
 
+var (
+	deprovisionJobsDurationHistogram = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "deprovision_jobs_duration_seconds",
+			Help: "Deprovision Jobs duration distribution",
+			//Buckets: []float64{0.5, 1, 2, 4, 8, 16},
+			Buckets: prometheus.ExponentialBuckets(1, 2, 9),
+		},
+		[]string{"uninstal_job", "namespace"},
```

Comment:
Do we want vectors on this one? This metric is used to calculated average uninstall times, but only triggers when it completes successfully. For this one I think we can go without the vector, but let me know if I'm thinking about it wrong.

Reply:
Actually I thought in the future we may have different namespaces that might have uninstall jobs, if that is not the case then we should not have a vector.

Reply:
I'm really not sure if they make sense here and I'm too new to prometheus, it would be one job per namespace in real world, does that make sense in a histogram? I assume we can average without vectors to get the real data we're after, but I'm not sure if we'd ever want to get an average for these buckets which would always probably be of size one. I think dropping them seems correct here.